src/Modules/addTask.js

```diff
@@ -0,0 +1,25 @@
+import { Tasks, updatingstorage } from './task.js';
+import sortOutput from './sort.js';
+
+const container = document.querySelector('.container');
+
+export default class Activity {
+  addTask() {
+    const submission = document.getElementById('submission');
+    const newArray = sortOutput();
+    const List = new Tasks();
+    List.description = submission.elements.addTolist.value;
+    List.index = `${newArray.length + 1}`;
+    List.completed = true;
+
+    newArray.push(List);
+
+    updatingstorage(newArray);
+    container.innerHTML = '';
+
+    // arrangement(newArray);
+    // removeTask();
+
+    return this;
+  }
```

Comment:
- [ ] Good job for Implementing  the a function for adding a new task (add a new element to the array) but as stated in the project requirement By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- [ ] So kindly make **the property completed set to false** and **when adding the new task the value of the array length adding in the reverse way as shown below** because when I add new task to UI , the task I first added display at the bottom of the UI but it should display at the First at the UI as shown below Kindly fix it.  
![Capture](https://user-images.githubusercontent.com/93598189/221171530-6bab43cb-4038-440a-b4ad-d69143e514c3.PNG)